web-app/src/app/api/cron/registry-sync/route.ts

```diff
@@ -45,23 +49,14 @@ export async function POST(request: Request) {
 const convertStatus = (
   status: "Online" | "Offline" | "Deregistered" | "Invalid",
 ) => {
-  return status.toUpperCase() as
-    | "ONLINE"
-    | "OFFLINE"
-    | "DEREGISTERED"
-    | "INVALID";
+  return status.toUpperCase() as "ONLINE" | "OFFLINE" | "DEREGISTERED";
```

Comment:
Isn't `INVALID` missing here?

Reply:
Sorry wrong commit, it was in https://github.com/masumi-network/sokosumi/pull/208/commits/eabc84b4ca92ea9a794f04dcce229ba2c8fe6f27